src/web/Fig.Web/Pages/Groups.razor.css

```
@@ -0,0 +1,694 @@
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    padding: 24px;
+    color: rgba(255, 255, 255, 0.45);
+    text-align: center;
+}
+
+::deep .groups-listbox {
```

Stylelint (17.7.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)